etc/user-data.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/ssh
+
+cat > /etc/newrelic-infra.yml <<EOF
+license_key: 001107cdaf3230d77288b1d1634cfe9dd669802c
+EOF
+service newrelic-infra restart
```

Comment:
Where is newrelic-infra RPM/deb file installed into the system?

Reply:
https://spinnaker.armory.io/#/applications/armoryhellodeploy/executions/configure/8187494b-2f79-4579-9482-f32220138ea5
In the bake stage, we bake in the newrelic-infra package.  The newrelic deb repo is configured in the extended attributes.